package.json

```diff
@@ -27,14 +27,38 @@
     "uuid": "^3.0.1"
   },
   "devDependencies": {
+    "eslint": "^2.13.1",
     "expect.js": "^0.3.1",
-    "gulp": "^3.9.1",
-    "gulp-eslint": "^2.1.0",
     "mocha": "^2.5.3",
     "sinon": "^1.17.7"
   },
   "scripts": {
-    "test": "gulp && mocha test/test_* --reporter spec --timeout 5000",
+    "lint": "eslint lib test",
```

Comment:
Can you have it ignore `lib/priority-queue.js`, as is done here: https://github.com/OptimalBits/bull/pull/515/files#diff-b9e12334e9eafd8341a6107dd98510c9R12